back/config/locales/nl-NL.yml

```diff
@@ -86,7 +86,7 @@ nl:
ify\">Hoe meld ik een aanstootgevend bericht?</h4><p class=\"ql-align-justify\">Klik wanneer je bent ingelogd op de drie kleine stippen rechts bovenaan het bericht om het te melden. Geef de reden aan waarom je de inhoud meldt - dit stuurt een melding naar de (project)beheerder die je melding gaat bekijken.</p><h4 class=\"ql-align-justify\">Waarom heb ik problemen met registreren of inloggen?</h4><p class=\"ql-align-justify\">Als je je niet kunt registreren, kan dat zijn omdat je al met hetzelfde e-mailadres geregistreerd bent. Log dan gewoon opnieuw in. Het kan ook zijn dat je niet kunt registreren omdat je eerder via e-mail al een uitnodiging kreeg om te registreren op het platform. Als je dus een bericht ziet dat je e-mailadres al in gebruik is omdat er al een uitnodiging in behandeling is, kijk dan in je e-mail en volg de link in de uitnodiging om je account te registreren. Als je de uitnodiging niet kunt vinden, probeer dan in je spambox te kijken of vraag eventueel aan een project- of platformbeheerder om een nieuwe uitnodiging te sturen. </p><h4 class=\"ql-align-justify\">Hoe kan ik mijn wachtwoord veranderen?</h4><p class=\"ql-align-justify\">Je kunt vanaf de inlogpagina altijd vragen om je wachtwoord te veranderen. Als je je wachtwoord opnieuw instelt, wordt een link naar je e-mailadres gestuurd. Volg de instructies om je wachtwoord te veranderen. Om veiligheidsredenen is deze link slecht één uur geldig. Als de link verlopen is, kun je op de inlogpagina opnieuw klikken op 'Wachtwoord vergeten?' om een nieuwe reset-link te genereren.</p><p class=\"ql-align-justify\">Als je de e-mail niet ontvangen hebt, controleer dan of deze echt niet in je spam-mail zit. Controleer ook zeker dat je de link stuurt naar hetzelfde e-mail adres als het e-mailadres waarmee je je eerder registreerde.</p><h4 class=\"ql-align-justify\">Wat gebeurt er met mijn gegevens?</h4><p class=\"ql-align-justify\">Je kunt het Privacybeleid lezen voor meer informatie over het verzamelen en gebruiken van gegevens. Kort samengevat: je gegevens worden nooit gebruikt voor reclame, marketing of andere vormen van spam.</p><h4 class=\"ql-align-justify\">Welke e-mails zal ik ontvangen?</h4><p class=\"ql-align-justify\">Je kunt automatische e-mails ontvangen gelinkt aan je account en je activiteit op het platform, bijvoorbeeld wanneer iemand reageert op één van jouw bijdragen of wanneer binnen een project een nieuwe fase start. Je kiest zelf welke automatische e-mails je van het platform ontvangt in je Instellingen. Daar is een rubriek \"Meldingen\" waar je e-mail-notificaties aan en uit kunt zetten.</p><h4 class=\"ql-align-justify\">Hoe verwijder ik mijn account?</h4><p class=\"ql-align-justify\">In je Instellingen is er de mogelijkheid om je account te verwijderen. Bijdragen die hebt geplaatst blijven zichtbaar, maar je naam wordt uit het bericht verwijderd en vervangen door 'onbekende auteur'. Als je al je bijdragen wilt verwijderen, doe dat dan voor je je account verwijdert.</p>"
```

Comment:
@jinjagit something went wrong here. Should still be "Veelgestelde vragen".

Reply:
@Brent Unfortunately, when I try to edit it back to the original it tells me that an existing identical translation has already been saved and I that i should vote for the existing translation. I have not found what that means or how to do it yet.